_inc/lib/class.media.php

```diff
@@ -110,15 +110,8 @@ static function get_allowed_mime_types( $default_mime_types ) {
 	 * @return bool
 	 */
 	protected static function is_file_supported_for_sideloading( $file ) {
-		if ( class_exists( 'finfo' ) ) { // php 5.3+
```

Comment:
Instead of getting rid of the PHP 5.3+ implementation, shouldn't we remove the old `mime_content_type` one? Or is that one better?

Reply:
I think `mime_content_type` is simpler than firing up the class, getting info out of it, then exploding out the first item listed.